Adyen/UI/Form/FormViewController.swift

```diff
@@ -183,7 +172,10 @@ open class FormViewController: UIViewController, Localizable, KeyboardObserver,
 
         resignFirstResponder()
 
-        showValidation()
+        /// Delay validation until after keyboard has disappeared
+        DispatchQueue.main.asyncAfter(deadline: .now() + .milliseconds(400)) { [weak self] in
```

Comment:
I don't think we have access to this value here

Reply:
can we add this to the `KeyboardObserver`? i think it would be much nicer to use the duration provided by the system, instead of a hardcoded one

Reply:
yes, if we put it in the `KeyboardObserver` we would be consuming the last frame the keyboard had the last time it showed, which might have been never (in case this is the first time the keyboard showed), so it would be race condition and unpredictable, so I would rather have a reasonable value here, if its off by a few milliseconds , then its should be perfectly fine!